pymc3/distributions/multivariate.py

```diff
@@ -1422,25 +1393,29 @@ def logp(self, x):
         -------
         TensorVariable
         """
-        n = self.n
-        eta = self.eta
-
-        X = x[self.tri_index]
+        _n = n.data
+        _eta = eta.data
+        shape = _n * (_n - 1) // 2
+        tri_index = np.zeros([_n, _n], dtype="int32")
+        tri_index[np.triu_indices(_n, k=1)] = np.arange(shape)
+        tri_index[np.triu_indices(_n, k=1)[::-1]] = np.arange(shape)
+
+        X = x[tri_index]
```

Comment:
This did 'fancy' indexing of the array `x` back in `v3`. Do we have an `Op` or a function which could handle it for `TensorVariable` in `v4`?